### PR TITLE
DEV-3182-new-go-exercise-fishandchips-fix

### PR DESCRIPTION
**Pull request by tiagocollot**
Added test/solution for go exam fishandchips.

### CODE REVIEW

File: solutions/fishandchips.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package solutions
+
+func FishAndChips(n int) string {
+	if n < 0 {
+		return "error: number is negative"
```
Comment (jotapero) [inline solutions/fishandchips.go:5]: Missing `\n` after the error

File: solutions/fishandchips.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	if (n%2 == 0) && (n%3 == 0) {
+		return "fish and chips\n"
+	}
+	if n%2 == 0 {
+		return "fish\n"
+	}
+	if n%3 == 0 {
+		return "chips\n"
+	}
+	return "error: non divisible"
```
Comment (jotapero) [inline solutions/fishandchips.go:17]: Missing `\n` after the error

File: solutions/fishandchips.go (hunk 1 above)
Comment (tiagocollot) [reply solutions/fishandchips.go:5]: It is fixed.

File: solutions/fishandchips.go (hunk 2 above)
Comment (tiagocollot) [reply solutions/fishandchips.go:17]: It is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

**Review by eslopfer — DISMISSED** [review]
Review pending of fix of description.

**Review by mikysett — APPROVED** [review]
LGTM

**Pull request merged**